vividus-plugin-excel/src/main/java/org/vividus/bdd/transformer/ExcelTableTransformer.java

```diff
@@ -59,33 +67,49 @@ public String transform(String tableAsString, TableParsers tableParsers, TablePr
             String lineBreakReplacementPropertyValue = properties.getProperties().getProperty("lineBreakReplacement");
             String lineBreakReplacement = lineBreakReplacementPropertyValue == null ? ""
                 : lineBreakReplacementPropertyValue;
-            result = extractData(excelSheetParser, properties)
+            List<String> result = extractData(excelSheetParser, properties)
                     .stream()
                     .map(e -> e.replace("\n", lineBreakReplacement))
                     .collect(Collectors.toList());
+            String columnName = properties.getProperties().getProperty(COLUMN);
+            if (columnName != null)
+            {
+                String joinValues = properties.getProperties().getProperty("joinValues");
+                notBlank(columnName, "Table property 'column' is blank");
+                headers = List.of(columnName);
+                data = List.of(Boolean.parseBoolean(joinValues) ? List.of(String.join(" ", result)) : result);
+            }
+            else
+            {
+                CellRangeAddress cellRangeAddress = CellRangeAddress.valueOf(properties.getProperties()
+                                                    .getProperty(RANGE));
+                int sizeColumn = cellRangeAddress.getLastColumn() - cellRangeAddress.getFirstColumn() + 1;
+                headers = result.subList(0, sizeColumn);
+                data = transformDataTable(headers, result.subList(sizeColumn,
+                        cellRangeAddress.getNumberOfCells()));
+            }
         }
         catch (WorkbookParsingException e)
         {
             throw new IllegalStateException("Error during parsing excel workbook", e);
         }
-        return build(result, properties);
+        return  ExamplesTableProcessor.buildExamplesTableFromColumns(headers, data, properties);
```

Comment:
the method becomes too long and looks too complicated, consider refactoring to improve readability